inlong-dashboard/src/metas/sources/defaults/index.ts

```diff
@@ -37,17 +37,17 @@ export const allDefaultSources: MetaExportWithBackendList<SourceMetaType> = [
     LoadEntity: () => import('./File'),
   },
   {
-    label: 'MySQL-BinLog',
-    value: 'MYSQL_BINLOG',
```

Comment:
The value here is to communicate with the manager. I think the value and file name should not be modified. Only the label configuration can be changed here.

Reply:
OK. I think it is necessary to maintain consistency. If it is not necessary, I will remove this change